### ISSUE TITLE
Video Collapses in RecyclerView

### ISSUE THREAD

**Issue by eygraber** [brianwernick/ExoMedia#124]
If I use `EmVideoView` in a `RecyclerView` calling `start()` plays the video, but it collapses the height to almost 0.

```java
  @Override
  public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
    final EmVideoView video = ...;
    video.setVideoURI(Uri.parse("http://www.sample-videos.com/video/mp4/480/big_buck_bunny_480p_10mb.mp4"));

    video.setOnClickListener(new View.OnClickListener() {
        @Override
        public void onClick(View v) {
          video.start();
        }
    });
  }
```

```xml
<?xml version="1.0" encoding="utf-8"?>

<FrameLayout
  xmlns:android="http://schemas.android.com/apk/res/android"
  android:layout_width="match_parent"
  android:layout_height="match_parent">

  <com.devbrackets.android.exomedia.ui.widget.EMVideoView
    android:id="@+id/video"
    android:layout_width="match_parent"
    android:layout_height="match_parent"/>
</FrameLayout>
```

**Comment by eygraber** [comment]
Hard-coding a height (e.g. `500dp`) prevents the video from collapsing, but this is not ideal. I'm not sure if this is an `ExoMedia` bug or a `RecyclerView` issue. Any thoughts?

**Comment by brianwernick** [comment]
Are you using the platform RecyclerView or the support version?  In the 23.2.0 support package they changed how sizing of children (ViewHolders) occurs.  

On another note, you should probably look at a previous question (https://github.com/brianwernick/ExoMedia/issues/13) about ViewHolder support.

**Comment by eygraber** [comment]
@brianwernick Updating the support library to 23.2.0 fixed it (they were ignoring `MATCH_PARENT` in the scroll direction in earlier versions). For anyone who can't upgrade to that, one workaround I found was to set the `LayoutParams.height` to the `RecyclerView` height in `onBindViewHolder`.

Thanks for pointing out #13; I've already disabled releasing when detaching from the window; working on some way to keep track of all the players so I can release them when they are no longer needed without leaking them.
